CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Reduced support for BitBox01
 - Fix a bug that would prevent the app to perform firmware upgrade when offline.
 - Replace sidebar with bottom navigation bar for mobile devices
+- Fixed "Send to self" button for non-BTC accounts.
```

Comment:
I'm not sure if this could be confusing for users as the send-to-self button is only shown in webdev

Reply:
yeah here: https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/6d3a224f12846480cd1c2a362333c0164cde332a/frontends/web/src/routes/account/send/components/inputs/receiver-address-input.tsx#L92